libs/designsystem/src/lib/components/modal/footer/modal-footer.component.ts

```diff
@@ -9,4 +9,8 @@ export class ModalFooterComponent {
   @HostBinding('class.snap-to-keyboard')
   @Input()
   snapToKeyboard = false;
+
+  @HostBinding('class.inline')
+  @Input()
+  inline = false;
```

Comment:
As this component might get extended with other types than `inline` (for example [`inline-sticky`](https://user-images.githubusercontent.com/65280082/101610909-22ece800-3a09-11eb-9f91-4b2add0a72f4.png)); i think we should call the input property `type` instead of `inline`. 

It should then have the following type: `type: 'inline' | 'fixed'` and default to `fixed`. 

This makes it possible to easily extend it with `inline-sticky` or similar in the future :)